### PR TITLE
Use typographic apostrophe instead of acute accent

### PR DESCRIPTION
**Pull request by lucaswerkmeister**
(Note: I haven’t tested this, so I’m not quite sure if it works. However, the acute accent is already outside 7-bit ASCII, and it worked, so I think it’s unlikely that the typographic apostrophe will break anything.)

### CODE REVIEW

File: app/views/Projects/edit.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
         <div class="controls">
             <input name="url" value="${flash.url ?: project.url}" maxlength="${util.Util.VARCHAR_SIZE}"/>
             <span class="help-inline">​#{error 'url'/}</span>
```
Comment (lucaswerkmeister) [inline app/views/Projects/edit.html]: This isn’t related to the pull request, but in the Git diff, I saw that there’s a zero-width space (U+200B) in this line between the `>` and the `#`. Same thing in `claimForm.html` just above. Is this intentional?

**Comment by lucaswerkmeister** [comment]
@FroMage comments?

**Comment by jvasileff** [comment]
Which is better: `let's` or `let’s`? They are both present after the patch, and I'd lean towards the first, since it's easier to type and keep consistent in a text editor, at least on my keyboard.

**Comment by lucaswerkmeister** [comment]
I’d say `let’s`, because it’s prettier, recommended by Unicode for this use, and it’s not the 70s anymore ;) but I’m biased since I use a keyboard layout that can type these characters, so they’re not as inconvenient for me as they are for others.

**Comment by lucaswerkmeister** [comment]
Note: [some argue](https://tedclancy.wordpress.com/2015/06/03/which-unicode-character-should-represent-the-english-apostrophe-and-why-the-unicode-committee-is-very-wrong/) that the English apostrophe should be represented by a different character altogether.

**Comment by jvasileff** [comment]
The 70s had it right. Like being able to simply type l instead of having to deal with the invention of the completely frivolous 1 key.

**Comment by gavinking** [comment]
Frankly I find `'` less jarring than most of the other options :-)

**Comment by lukedegruchy** [comment]
'

**Comment by FroMage** [comment]
The article about the two apostrophes does have a good point, but I wonder how both options are affected by word breaking? It wouldn’t do do have breaks in the middle of words.

**Comment by lucaswerkmeister** [comment]
Let’s try it out:

- U+0027 APOSTROPHE
  > aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'aaa'
- U+2019 RIGHT SINGLE QUOTATION MARK
  > aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’aaa’
- U+02BC MODIFIER LETTER APOSTROPHE
  > aaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼaaaʼ

Looks like none of them allow a word break.

**Comment by Zambonifofex** [comment]
@lucaswerkmeister

If I remember from when I read the article (it’s been a while), it talks about how it affects the word break when the apostrophe is *at the end* or *at the beginning* of a word (for when double‐clicking, for example). Consider:

There were multiple cars.

U+0027 APOSTROPHE
The cars' windows were broken.

U+2019 RIGHT SINGLE QUOTATION MARK
The cars’ windows were broken.

U+02BC MODIFIER LETTER APOSTROPHE
The carsʼ windows were broken.

Honestly, I think using U+2019 is a little bit puristic, the U+0027 conveys the same meaning. I do use U+2019 because the Unicode Consortium says so, but I’ll admit I’m a bit puristic myself (I’d get bothered if someone said “There ***was*** multiple cars.”); I’m the only person I know that goes to the trouble of hitting <samp>ctrl+shit+u</samp> and then typing “2010” to write a hyphen or “2212” to write a minus sign.

**Comment by FroMage** [comment]
This PR has been closed as a result of moving Ceylon to the Eclipse Foundation, not as a judgement of its value. We are unfortunately unable to restore it ourselves, but if you feel this PR should be reopened, please rebase your work on the current master and create a new PR. Thanks, and sorry for the inconvenience.

**Comment by lucaswerkmeister** [comment]
So the result of some arguing about which proposed solution is better is that we haven’t merged any solution and still use backticks, which I don’t think anyone has argued for :/

Does anyone care about this enough to pick it up again?

**Pull request closed without merging**